src/tokens/safe.tokenlist.json

```diff
@@ -71,6 +71,22 @@
       "decimals": 18,
       "chainId": 100,
       "logoURI": "https://assets.coingecko.com/coins/images/31099/standard/OLAS-token.png"
+    },
+    {
+      "symbol": "cUSDCv3",
+      "name": "Compound USDC",
+      "address": "0xc3d688b66703497daa19211eedff47f25384cdc3",
+      "decimals": 18,
+      "chainId": 1,
+      "logoURI": "https://safe-transaction-assets.safe.global/tokens/logos/0xc3d688B66703497DAA19211EEdff47f25384cdc3.png"
```

Comment:
Not sure if this logo exists. Do I need to add it somewhere?

Reply:
We can ask tx-service to add it.